CONTRIBUTING.md

```diff
@@ -306,9 +313,28 @@ $ ./cli exec --authn-oidc
 root@9feae5e5e001:/src/conjur-server#
 ```
 
+###### Run Cukes with Google Cloud Platform (GCP) Compatible Environment
```

Comment:
I am a bit confused why this is titled "Run Cukes with Google Cloud Platform (GCP) Compatible Environment" if it is never really shown how to run GCP. Here we _only_ show how to setup the environment so the user can run the tests. Am I correct?

This applies for "Run Cukes with Open ID Connect (OIDC) Compatible Environment" as well

Reply:
If my understanding is correct, consider "Spin up Google Cloud Platform (GCP) Compatible Environment for testing". WDYT?

Reply:
I'll omit the WDYT

Reply:
preferably :P